pages/common/nrm.md

```diff
@@ -0,0 +1,28 @@
+# nrm
+
+> JavaScript and Node.js package registry manager.
+> Nrm can help you easy and fast switch between different npm registries, now include: npm, cnpm, taobao, nj(nodejitsu), rednpm.
+
+- List all the registries:
+
+`nrm ls`
+
+- Change registry to registry:
+
+`nrm use {{registry}}`
+
+- Show the response time for one or all registries:
+
+`nrm test [{{registry}}]`
+
+- Add one custom registry:
+
+`nrm add {{registry}} {{url}} [{{home}}]`
```

Comment:
Are the square brackets here required?

Reply:
Doesn't seem to be required in any of the examples, looking at the man page these are just denoting optional arguments.

Reply:
Ah, ok